vnext/Shared/JSI/RuntimeHolder.h

```diff
@@ -6,6 +6,8 @@
 namespace facebook {
 namespace jsi {
```

Comment:
(nit) maybe update to modern namespace syntax? :)
namespace facebook::jsi {

Reply:
Not only that! If this is not the implementation of a Facebook API, we should probably rename it to `Microsoft::JSI`.